### PR TITLE
DEV 011 creating admin page

### PR DESCRIPTION
**Pull request by Paul-Gerarts**

### CODE REVIEW

File: src/main/java/be/syntra/devshop/DevshopFront/models/StatusNotification.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
     PASSWORD_NO_MATCH("The provided passwords don't match!"),
     REGISTER_FAIL("Registering new user failed. Try a different emailaddress"),
+    UPDATED("The update was succesful!"),
```
Comment (lenshuygh) [inline src/main/java/be/syntra/devshop/DevshopFront/models/StatusNotification.java:12]: mind the spelling of 'successful' ;-)

File: src/main/java/be/syntra/devshop/DevshopFront/models/StatusNotification.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
...
     SUCCES("Succes"),
```
Comment (lenshuygh) [inline src/main/java/be/syntra/devshop/DevshopFront/models/StatusNotification.java:13]: 'success'

File: src/main/java/be/syntra/devshop/DevshopFront/models/StatusNotification.java (hunk 1 above)
Comment (Paul-Gerarts) [reply src/main/java/be/syntra/devshop/DevshopFront/models/StatusNotification.java:12]: https://www.spellchecker.net/misspellings/succesfull

File: src/main/java/be/syntra/devshop/DevshopFront/models/StatusNotification.java (hunk 2 above)
Comment (Paul-Gerarts) [reply src/main/java/be/syntra/devshop/DevshopFront/models/StatusNotification.java]: Fixed

**Pull request merged**